app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -95,15 +99,6 @@
         box-shadow: none;
       }
 
-      .contents-number {
-        float: left;
-      }
-
-      .contents-text {
-        display: block;
-        margin-left: 1.5em;
-      }
-
       .direction-rtl & {
         .contents-number {
           float: right;
```

Comment:
Fixed in 64bf8f5

![screen shot 2017-07-03 at 14 01 13](https://user-images.githubusercontent.com/319055/27794150-cfc3c22c-5ff8-11e7-9a84-e10048d551cc.png)

https://www.gov.uk/government/publications/why-overseas-companies-should-set-up-in-the-uk/1651201